joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreator.scala

```diff
@@ -1863,31 +1869,16 @@ class AstCreator(filename: String, global: Global)
       methodNameAsIdentifierStack.push(methodIdentifierAsts.head)
       val argsAsts = astForArgumentsWithoutParenthesesContext(ctx.argumentsWithoutParentheses())
 
-      val callNodes = methodIdentifierAsts.head.nodes.filter(node => node.isInstanceOf[NewCall])
+      val callNodes = methodIdentifierAsts.head.nodes.collect(_.asInstanceOf[NewCall])
```

Comment:
```suggestion
      val callNodes = methodIdentifierAsts.head.nodes.collectAll[NewCall]
```

Reply:
there is no option as `collectAll`, only `collect` and `collectFirst` are there.